kuma/javascript/src/ssr.jsx

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import { renderToString } from 'react-dom/server';
 
 import App from './app.jsx';
+import { localize } from './l10n.js';
 
 /*
  * This function performs server-side rendering of our UI, given
  * a JSON object of document data. It is used by ../ssr-server.js
  */
 export default function ssr(data) {
-    return renderToString(
-        <App documentData={data.documentData} requestData={data.requestData} />
-    );
+    // Store the string catalog so that l10n.gettext() can do translations
+    localize(data.requestData.locale, data.localizationData);
```

Comment:
Like before, I would vote for adding `getLocale` to the import from `./l10n.js` above, use `getLocale()` instead of `data.requestData.locale` in the call to `localize`, and get rid of `requestData` altogether.